components/identities/actions/identityActions.tsx

```
@@ -319,21 +318,72 @@ const IdentityActions: FunctionComponent<IdentityActionsProps> = ({
+                    <ClickableAction
+                      title="ENABLE SUBSCRIPTION"
+                      description={nextAutoRenew}
+                      style="primary"
+                      icon={<div className={styles.renewalIcon}>ON</div>}
+                      onClick={() => router.push("/subscription")}
+                    />
+                  ) : null}
+
+                  {/* This is the the old  */}
+                  {callDataEncodedDomain[0] === "1" && false &&
+                    isAutoRenewalEnabled ? (
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)